apps/site-app/src/containers/Accounts/BillPayments/payees.ts

```diff
@@ -19,7 +19,7 @@ const translations = defineMessages({
 });
 
 export type Validatable = {
-  validate: (value: string) => ValuedMessageDesc|null,
```

Comment:
Also had a type define with the same purpose, so merged them together for both UxInput & ModalOperation